source/content/multidev-faq.md

```diff
@@ -8,12 +8,12 @@ For information about what Multidev is and how to use it, see our full guide on
 
 ## Who has access to Multidev?
 
-Multidev is available to:
+Multidev is available to all **accounts with [Gold support](/support/#support-features-and-response-times) and above**. This includes:
 
- - Enterprise organizations.
- - EDU organizations with [Gold support](/support/#support-features-and-response-times) and above.
- - Pantheon Partner Program agencies (at the level of Pantheon Partner and above).
- - Direct Online customers with [Gold support](/support/#support-features-and-response-times) and above.
+ - Enterprise organizations
+ - EDU organizations
+ - Pantheon Partner Program agencies (at the level of Pantheon Partner and above)
+ - Direct Online customers
```

Comment:
@EdwardAngert not *all* direct online customers have gold support, but this could be read as saying so, what do you think?

Reply:
@EdwardAngert This can also be read as the 4 account types listed are those that have Gold support and thus is available—which is not what is intended.  Trying to convey that to qualify for Multidev you _must_ have one of the following account types AND Gold or higher support. Silver support, no matter the account type, does not include multidev.